virtualization-and-containers/material/containers_slides.md

```diff
@@ -0,0 +1,105 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Containers
+
+---
+
+## What is a Container?
+
+<img src="https://raw.githubusercontent.com/Simulation-Software-Engineering/Lecture-Material/main/virtualization-and-containers/material/figs/container-sketch.png" width=60%; style="margin-left:auto; margin-right:auto; padding-top: 25px; padding-bottom: 25px">
+
+
+- Container operates in "fenced off" part of the operating system (`namespaces`)
+- Low(er) overhead than virtual machines
+  - Runs on kernel (and libraries) of the host OS
+  - Cheap to start and stop a container
+- Available features depend on Host
+  - Linux container on Windows or vice versa?
+- Container (can be) encapsulated
```

Comment:
why it brackets? does it make sense without brackets?

Reply:
I don't know why the parenthesis are ther. I remove them.